### PR TITLE
migrate tenant id out of event repoID

### PR DESCRIPTION
**Pull request by ryanfkeepers**
## Description

Moves the tenantID out of the repoID hash and
into its own value (also hashed) in the event log.

Unique user identification to follow.

## Type of change

- [x] :sunflower: Feature

## Issue(s)

* #1602

## Test Plan

- [x] :green_heart: E2E

### CODE REVIEW

File: src/internal/events/events.go [hunk 1]
```diff
@@ -117,6 +118,7 @@ func (b Bus) Event(ctx context.Context, key string, data map[string]any) {
 	props := analytics.
 		NewProperties().
 		Set(repoID, b.repoID).
+		Set(tenantID, b.tenant).
```
Comment (gmatev) [inline src/internal/events/events.go:121]: `tenantID` is also needed as a Trait in the Identify call

**Review by gmatev — CHANGES_REQUESTED** [review]
`tenantID` is also needed as a Trait in the Identify call

File: src/internal/events/events.go (hunk 1 above)
Comment (ryanfkeepers) [reply src/internal/events/events.go:121]: Okay, will re-add.

**Comment by gmatev** [comment]
@ryanfkeepers I think my initial comment was misleading. Having both IDs as event dimensions is good by my point was for the actual identify code. Was not sure who to leave a comment in part that has not changed. You want something like 

```
err := b.client.Enqueue(analytics.Identify{
			UserId: b.repoID,
			Traits: analytics.NewTraits().
				SetName(b.repoID),
                                 Set(tenantID, b.tenant),
		})
```

**Review by gmatev — APPROVED** [review]
I think this is good for this PR. We should file a ticket for a follow-up that captures the work needed to have a repoId based on an actual unique per repo identifier

**Pull request merged**